app/repositories/sensor/sensor.repository.go

```diff
@@ -0,0 +1,141 @@
+package sensor
+
+import (
+	"context"
+
+	model "database-ms/app/models"
+	"database-ms/config"
+
+	mgo "gopkg.in/mgo.v2"
+	"gopkg.in/mgo.v2/bson"
+)
+
+// SensorRepository, used to perform DB operations
+// Interface contains basic operations on sensor document
+// So that, DB operation can be performed easily
```

Comment:
Find one by Id? Which ID, what is one? Difference between findOne?

 IsSensorAlreadyExists? -> Exists